<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-LDEJGU9" failures="2" tests="6" name="Test" time="52.278" errors="0" timestamp="2021-06-28T01:34:23 IST">
  <testcase classname="org.def.Thirdone" name="testcase2" time="0.045"/>
  <testcase classname="org.def.Thirdone" name="testcase4" time="0.011">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at org.def.Thirdone.testcase4(Thirdone.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testcase4 -->
  <testcase classname="org.def.Thirdone" name="testcase6" time="0.002"/>
  <testcase classname="org.def.Secondone" name="testcase1" time="0.012">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at org.def.Secondone.testcase1(Secondone.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testcase1 -->
  <testcase classname="org.def.Secondone" name="testcase3" time="0.004"/>
  <testcase classname="org.def.Secondone" name="testcase7" time="0.01"/>
</testsuite> <!-- Test -->
